NewHorizons/Main.cs

```diff
@@ -79,15 +80,18 @@ public override void Configure(IModConfig config)
 
             var currentScene = SceneManager.GetActiveScene().name;
 
-            Debug = config.GetSettingsValue<bool>("Debug");
+            Debug = config.GetSettingsValue<bool>("Debug");
+            VerboseLogs = config.GetSettingsValue<bool>("Verbose Logs");
 
             if (currentScene == "SolarSystem")
             {
                 DebugReload.UpdateReloadButton();
                 DebugMenu.UpdatePauseMenuButton();
             }
 
-            Logger.UpdateLogLevel(Debug ? Logger.LogType.Log : Logger.LogType.Error);
+            if (Debug && VerboseLogs) Logger.UpdateLogLevel(Logger.LogType.Verbose);
```

Comment:
This would make it so if `Debug` is false and `Verbose logs` is true, log level would still be `LogType.Error`

Reply:
corby did it